src/PagePropsStatementCountLookup.php

```diff
@@ -14,13 +14,15 @@ public function __construct( LoadBalancer $loadBalancer ) {
 	}
 
 	public function getStatementCount( EntityId $entityId ) {
+		global $wgWBRepoSettings;
```

Comment:
I’ve changed it now to use `WikibaseRepo::getEntityNamespaces()` instead of accessing the global variable directly. The `WikibaseRepo` class is already used several times in the importer (and always with `getDefaultInstance()` as far as I can tell).